api/helpers/gamestate/get-log.js

```diff
@@ -59,14 +59,14 @@ module.exports = {
         case MoveType.JACK:
           return `${player} stole ${opponent}'s ${targetCardName} with the ${playedCardName}.`;
 
-        case MoveType.UNTARGETED_ONE_OFF:
-          return `${player} played the ${playedCardName} as a one-off to 
+        case MoveType.ONE_OFF: {
```

Comment:
Combined `MoveType.UNTARGETED_ONE_OFF` and `MoveType.TARGETED_ONE_OFF` into a single `MoveType.ONE_OFF`